pkg/expr/sql/db.go

```diff
@@ -15,6 +16,42 @@ import (
 // DB is a database that can execute SQL queries against a set of Frames.
 type DB struct{}
 
+// GoMySQLServerError represents an error from the underlying Go MySQL Server
+type GoMySQLServerError struct {
```

Comment:
Wasn't sure if I was over-engineering, adding this now when we could add it later. But I lean towards keeping this implementation - it feels pretty rock solid.